packages/chips/Chip.tsx

```diff
@@ -134,12 +134,38 @@ export class Chip extends React.Component<ChipProps, ChipState> {
           .getPropertyValue(propertyName);
       },
       getRootBoundingClientRect: () => {
-        if (!this.chipElement) return new ClientRect();
+        if (!this.chipElement) {
+          // new DOMRect is not IE11 compatible
+          const defaultDOMRect = {
+            bottom: 0,
+            height: 0,
+            left: 0,
+            right: 0,
+            top: 0,
+            width: 0,
+            x: 0,
+            y: 0,
+          };
+          return defaultDOMRect;
+        }
         return this.chipElement.getBoundingClientRect();
       },
       getCheckmarkBoundingClientRect: () => {
         const {chipCheckmark} = this.props;
-        if (!chipCheckmark) return new ClientRect();
+        if (!chipCheckmark) {
```

Comment:
@moog16 Should this `if` also check `chipCheckmark.props.getBoundingClientRect`?

Reply:
Yes, that doesn't hurt to have the check there.

Reply:
I won't merge it if you want to update this...you may also want to check unit tests.

Reply:
Added new commit.

Couldn't run tests due to fsevents failing to build on Node v12 on Mac.

I'll try to see about switching to nvm tomorrow and running the tests.